src/views/token-swap/token-swap.view.jsx

```diff
@@ -119,7 +144,9 @@ const mapDispatchToProps = dispatch => ({
     dispatch(globalActions.changeHeader(getHeader(currentStep))),
   onGoToQuotes: () => dispatch(tokenSwapActions.goToQuotes()),
   onCleanup: () => dispatch(tokenSwapActions.resetState()),
-  onLoadAccounts: fromItem => dispatch(tokenSwapThunks.fetchAccounts(fromItem))
+  onLoadAccounts: fromItem => dispatch(tokenSwapThunks.fetchAccounts(fromItem)),
+  onLoadingQuotes: () => dispatch(tokenSwapActions.getQuotes()),
```

Comment:
We don't need this handler as we are already dispatching this function from the `getQuotes` thunk.

Reply:
We need it because we launch loading spinner at the moment that you write but action from thunks after timeout

Reply:
But that's not true then as we are not loading any quotes. It leads to confusion. I would create another variable to handle the state of the spinner, for example: `showSpinner` which should be enabled if `quotes.status === "loading" || userIsTyping` (just some pseudocode to understand what I'm trying to explain).